modules/sdk-coin-sui/src/lib/transactionBuilder.ts

```diff
@@ -91,12 +91,6 @@ export abstract class TransactionBuilder extends BaseTransactionBuilder {
     return this;
   }
 
-  gasPrice(gasPrice: number): this {
```

Comment:
Removal of publicly exposed method in the stable release requires a BREAKING CHANGE: description in the commit footer.

Reply:
Got it, added the description!
Since Sui is still in progress and not announced to the public, so it probably won't cause too much trouble.